<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDialog</class>
 <widget class="QDialog" name="OutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>564</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Facial Recognition</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon1.png</normaloff>icon1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>611</width>
     <height>451</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>611</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Buttons">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ClockInButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/book-download.png</normaloff>Icons/book-download.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ClockOutButton">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/book-upload.png</normaloff>Icons/book-upload.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Capture">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/aperture.png</normaloff>Icons/aperture.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Switch">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/power.png</normaloff>Icons/power.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="DateTimeBox">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Date">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Date_label">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Time">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Time :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Time_label">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="Details">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>100</y>
     <width>271</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Details</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="Name">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NameLabel">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="Status">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Class :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="StatusLabel">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="Checked">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Status :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="HoursLabel">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="MinLabel">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="SlideMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>541</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>326</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QToolBox" name="SideMenuBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>211</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset>
       <normaloff>Icons/id.png</normaloff>Icons/id.png</iconset>
     </attribute>
     <attribute name="label">
      <string>Profiles</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="Student">
         <property name="text">
          <string>Student</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/school.png</normaloff>Icons/school.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>211</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset>
       <normaloff>Icons/clipboard-text.png</normaloff>Icons/clipboard-text.png</iconset>
     </attribute>
     <attribute name="label">
      <string>Attendance</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="viewbtn">
         <property name="text">
          <string>View</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>211</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset>
       <normaloff>Icons/help.png</normaloff>Icons/help.png</iconset>
     </attribute>
     <attribute name="label">
      <string>About</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>171</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>This Software was created by</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Team Code Aethers</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>as a requirement</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>to the completion of</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Capstone Research</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>from the institution of</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>City College of Calamba</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
